src/views/3-Modeling/Modeling.vue

```diff
@@ -916,268 +921,191 @@
-                .attr("cx", function(d) { return self.xScale(d.ANN)})
-                .call(enter => enter.transition(t)),
-              update => update  
-                .attr("cx", function(d) { return self.xScale(d.ANN)})
-                .call(update => update.transition(t))
-
-            )
-
           },
-          //update bee x position on scroll
-          // this is done by changing the forceX to a different x variable
-          updateChart(data, data_step) {
+          addBees(step_in, data_var) {
             const self = this;
 
-            //modify the force depending on step
-            // list models in order of transitions, use step index to select
-            var model_sel = this.model_list[data_step];
-
-            // stop previous simulation
-           //this.force_sim.stop()
-          
-          //move all points to center when model is introduced?
-              if (data_step === 2){
-
-              console.log(model_sel);
-
-                  self.force_sim
-                    .force('x', this.d3.forceX(500).strength(.4))
-                    .force('y', this.d3.forceY(500).strength(.4))
-                    .alpha(0.2)
-                    .alphaDecay(0.05)
-                  
-              } else {
-
-            console.log(model_sel);
-
-            // move points along x axis according to model, when "ticked"
-            // this should only happen if the model is different than the previous
-
-            this.force_sim
-              .force('x', this.d3.forceX(function(d){
-                return self.xScale(d[model_sel])
-              }).strength(1))
-              .alpha(0.5)
-
-              }
-              // reheat the simulation to make thigns move
-              // charge or "heat" is defined by alpha, 
-              this.force_sim.restart()
-                .on("tick", self.tick)
-
-              this.init_decay = setTimeout(function(){
-                console.log('re-init alpha decay');
-                this.force_sim.alphaDecay(0.01);
-              }, 500)
-              clearTimeout(this.init_decay);
+          console.log(this.chartState.measure);
+
+        // define and stop sim
+        // need to make a different function for the initial force draw, because will want to defien
+        // where things are coming from differently?
+        self.simulation = this.d3.forceSimulation(this.chartState.dataset, function(d) { return d.seg })
+          .force("x", this.d3.forceX((d) => this.xScale(d[this.chartState.measure])).strength(1))
+          .force('y', this.d3.forceY(this.height/2).strength(0.3))
+          .force("collide", this.d3.forceCollide(this.paddedRadius).strength(.9).iterations(10));
+
+          // bind data
+          let chart = this.svg.selectAll(".bees") // puts out error on intial draw until scrolled
+          .data(this.chartState.dataset, function(d) { return d.seg }) // use seg as a key to bind and update data
```

Comment:
🎉Nice work figuring out this enter exit update situation **AND** all the force simulation variables ✨  I definitely can't follow all of this, but I'm looking forward to walking through it at some point!